packages/story-editor/src/components/form/color/color.js

```diff
@@ -52,7 +52,8 @@ const containerCss = css`
 const Container = styled.section`
   ${containerCss}
   gap: ${({ isInDesignMenu }) => (isInDesignMenu ? 6 : 8)}px;
-  width: ${({ width }) => (width ? `${width}px` : `100%`)};
+  width: ${({ width, ignoreWidth }) =>
+    (ignoreWidth && 'inherit') || (width ? `${width}px` : `100%`)};
```

Comment:
🤔  So this feels a little strange to me. The `Color` component takes a `width` prop, but if there's a "multiple" value then the component needs to ignore the passed in width.

I tried removing lines 55 and 56, as well as removing lines 145 (so that `width` isn't passed to the div. Seems to me like everything worked as intended? The color input looks the same in the floating menu and in the style pane.

It seems to me like letting the container decide its own width would work here, but lmk if I'm missing a case.

Reply:
no I think you're right. I think the width stuff within the floating menu is a little wonky. I think it's there for calculating the bounds of where we can place it, but then we also explicitly set it? I'm not sure what the purpose of explicitly setting the width is. also feels diffferent to need all the mutation observers we have there to gather this info, but I think I'm lacking some context 🤷

Reply:
>I tried removing lines 55 and 56, as well as removing lines 145 (so that width isn't passed to the div. Seems to me like everything worked as intended? The color input looks the same in the floating menu and in the style pane. Without exciplty setting it, the flex extends both containers where color is used. So we end up with extra wide `more` button and and extra wide opacity input. 

But, we could still use `inherit` instead of the extra prop. 

something like 
```
  width: ${({ width }) => (width ? `${width}` : `inherit`)};

```
Where `width` is string that contains units. eg. `width = "128px"`  or `width='inherit"` depend on if it is mixed or not. 

<img width="788" alt="Screen Shot 2022-03-24 at 9 39 23 AM" src="https://user-images.githubusercontent.com/1820266/159955193-685ea859-4719-47a0-a6c4-25a60dc594ec.png">

Reply:
ok, i think we're good here - removed the `100%` fallback and need for an extra prop.